microcosm/loaders/__init__.py

```diff
@@ -32,5 +27,9 @@ def load_from_dict(dct=None, **kwargs):
     dct.update(kwargs)
 
     def _load_from_dict(metadata):
-        return dict(dct)
+        return Configuration(dct)
     return _load_from_dict
+
+
+def empty_loader(metadata, Metadata):
+    return Configuration()
```

Comment:
I've had use for this in various places, esp as a default value for an `extra_loader` parameter to `create_app`